tripy/tripy/frontend/trace/ops/expand.py

```diff
@@ -29,6 +29,7 @@
 @dataclass(repr=False)
 class Expand(BaseTraceOp):
     output_rank: int
+    output_len: Optional[int]
```

Comment:
Does this mean anything for non-shape tensors? Is it set in that case? Could you document the behavior in a comment here?

Reply:
No, it should not be used otherwise. I will add a comment